src/components/particles/ColorVariant.vue

```diff
@@ -1,13 +1,15 @@
 <template>
   <div
-    class="color-variant mr-2 my-1"
-    :style="{ backgroundColor: colorHex }"
+    class="rounded-lg mr-2"
+    :style="{ backgroundColor: colorHex, height: '40px', width: '80px' }"
```

Comment:
@balanejacob You don't need to put these in inline styles, you can use the `style` tag instead https://vuejs.org/api/sfc-css-features.html#scoped-css